components/layout/section/section.js

```diff
@@ -1,38 +1,37 @@
 import PropTypes from 'prop-types'
-import { Box, Grid, Typography } from '@mui/material'
+import { Box, Typography, Stack } from '@mui/material'
 
 export const Section = ({ title, children }) => {
   return (
-    <Grid
-      container
-      columns={ 8 }
-      columnSpacing={{ xs: 0, md: 2 }}
-      rowSpacing={{ xs: 2, md: 10 }}
-      sx={{
-        margin: '3rem 0',
+    <Stack
+    direction={{ sm: 'column', md: 'row' }}
+    spacing={{ sm: 2, md: 6 }}
+    sx={{
+        margin: '3rem 6rem',
         '& .title': {
           xs: { textAlign: 'left' },
           md: { textAlign: 'right' },
         },
       }}
     >
-      <Grid item xs={ 8 } md={ 3 }>
+      <Box sx={{
```

Comment:
with a `flex-grow: 0` (or `flex: 0 ...`) the right-hand side won't grow to fill the available space, so try something like this (unsure about MUI/sx syntax :smile:  )

```
      <Box sx={{
        flex: {
          xs: '0 0',
          md: '0 1 200px',
        }
      }}>
```

Reply:
maybe '0 0 200px' ?